cynic/src/core.rs

```diff
@@ -37,6 +37,18 @@ where
     }
 }
 
+impl<T> QueryFragment for MaybeUndefined<T>
+where
+    T: QueryFragment,
+{
+    type SchemaType = Option<T::SchemaType>;
+    type VariablesFields = T::VariablesFields;
+
+    fn query(builder: SelectionBuilder<'_, Self::SchemaType, Self::VariablesFields>) {
+        T::query(builder.into_inner())
+    }
+}
```

Comment:
I'm not really sure is there's a use case for MaybeUndefined as a QueryFragment.  It makes sense for input types but not so much for output types which will never be undefined in a well formed graphql response....